src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml

```diff
@@ -23,98 +27,98 @@
+            @if (!Model.Report.Submitted)
+            {
+                if (Model.CanBeEdited)
                 {
-                    if (Model.CanBeEdited)
-                    {
-                        <a class="link-back" asp-controller="Report" asp-action="Edit">Back</a>
-                    }
-                    else
-                    {
-                        <a class="link-back" asp-controller="Home" asp-action="Index">Back</a>
-                    }
-
+                    <a class="link-back" asp-controller="Report" asp-action="Edit">Back</a>
                 }
                 else
                 {
-                    <a class="link-back" asp-controller="Report" asp-action="List">Back</a>
+                    <a class="link-back" asp-controller="Home" asp-action="Index">Back</a>
                 }
 
+            }
+            else
+            {
+                <a class="link-back" asp-controller="Report" asp-action="List">Back</a>
+            }
+
 
-                @if (Model.Report.Submitted)
+            @if (Model.Report.Submitted)
+            {
+                <legend>
+                    <h1 class="heading-xlarge">Submitted details</h1>
+                </legend>
+            }
+            else
+            {
+                if (Model.IsReadOnly)
                 {
                     <legend>
-                        <h1 class="heading-xlarge">Submitted details</h1>
+                        <h1 class="heading-xlarge">View details</h1>
                     </legend>
+                    <p style="white-space: pre-line">@Model.Subtitle</p>
                 }
                 else
                 {
-                    if (Model.IsReadOnly)
-                    {
-                        <legend>
-                            <h1 class="heading-xlarge">View details</h1>
-                        </legend>
-                        <p style="white-space: pre-line">@Model.Subtitle</p>
-                    }
-                    else
-                    {
-                        <legend>
-                            <h1 class="heading-xlarge">Review details</h1>
-                        </legend>
-                        <p style="white-space: pre-line">@Model.Subtitle</p>
-                    }
-                    <div asp-validation-summary="All" class="error-summary" role="alert" aria-labelledby="error-summary-heading" tabindex="-1">
-                        <h2 class="heading-medium error-summary-heading" id="error-summary-heading">
-                            @(Model.IsReadOnly ? "These are the current errors" : "Please fix these errors")
-                        </h2>
-                    </div>
+                    <legend>
+                        <h1 class="heading-xlarge">Review details</h1>
+                    </legend>
+                    <p style="white-space: pre-line">@Model.Subtitle</p>
                 }
-
-                <div class="form-group">
```

Comment:
what does this change have to do with printing the page?

Reply:
Github weirdness no changes made